.circleci/config.yml

```diff
@@ -1,65 +1,142 @@
 version: 2
 jobs:
   lint:
-    machine: true
+    docker:
+      - image: quay.io/deis/lightweight-docker-go:v0.2.0
+    environment:
+      SKIP_DOCKER: true
+      CGO_ENABLED: 0
+    working_directory: /go/src/github.com/Azure/open-service-broker-azure
     steps:
       - checkout
+      # This is a workaround for a problem I (krancour) cannot pin down. The
+      # linter works flawlessly in all environments (containers included)
+      # EXCEPT a container in Circle. I've remoted into running containers in
+      # Circle and have attempted, at great length, to diagnose the issue, but
+      # am unable to do so. What I did discover, however, was that by
+      # re-installing gometalinter (and the linters it, in turn, uses), I can
+      # make the problem go away. ODDLY, when comparing the binaries that were
+      # included in the image to the NEW binaries, diff says they are identical.
+      # I'm at a complete loss for why this happens. I can live with this
+      # workaround for now because I anticipate our days using Circle for CI
+      # are numbered anyhow.
```

Comment:
I spent nearly a whole day trying to pin down the cause of this. I give up. This workaround works. I couldn't tell you why.